spreadsheet_oca/static/src/spreadsheet/pivot_controller.esm.js

```diff
@@ -0,0 +1,67 @@
+import {PivotRenderer} from "@web/views/pivot/pivot_renderer";
+import {_l} from "@web/core/l10n/translation";
+import {patch} from "@web/core/utils/patch";
+
+patch(PivotRenderer.prototype, {
+    isComparingInfo() {
+        return Boolean(this.model.searchParams.comparison);
+    },
+    containsDuplicatedGroupBys() {
+        const colGroupBys = new Set(
+            this.model.metaData.colGroupBys
+                .concat(this.model.metaData.expandedColGroupBys)
+                .map((el) => el.split(":")[0])
+        );
+        const rowGroupBys = new Set(
+            this.model.metaData.rowGroupBys
+                .concat(this.model.metaData.expandedRowGroupBys)
+                .map((el) => el.split(":")[0])
+        );
+        return Boolean(colGroupBys.intersection(rowGroupBys).size);
+    },
+    containsColGroupBys() {
+        const colGroupBys = new Set(
+            this.model.metaData.colGroupBys
+                .concat(this.model.metaData.expandedColGroupBys)
+                .map((el) => el.split(":")[0])
+        );
+        return Boolean(colGroupBys.size);
+    },
+    disableSpreadsheetInsertion() {
+        return (
+            !this.model.hasData() ||
+            !this.model.metaData.activeMeasures.length ||
+            this.containsDuplicatedGroupBys() ||
+            this.isComparingInfo()
+        );
+    },
+    getSpreadsheetInsertionTooltip() {
+        var message = _l("Add to spreadsheet");
+        if (this.containsDuplicatedGroupBys()) {
+            message = _l("Duplicated groupbys in pivot are not supprted");
+        } else if (this.isComparingInfo()) {
+            message = _l("Comparisons in pivot are not supprted");
```

Comment:
To fix @tarteo error, you can do this

```suggestion
        var message = _t("Add to spreadsheet");
        if (this.containsDuplicatedGroupBys()) {
            message = _t("Duplicated groupbys in pivot are not supprted");
        } else if (this.isComparingInfo()) {
            message = _t("Comparisons in pivot are not supprted");
```

After updating the import
```
import {_t} from "@web/core/l10n/translation";
```

Reply:
@TheToto42 Thanks, I will add it now :)